### ISSUE TITLE
MemoServ memo treated as welcome message

### ISSUE THREAD

**Issue by nyuszika7h** [weechat/weechat#473]
Someone sent me a memo containing `[#channel] message` and it ended up as a PvNotice in `#channel`. Maybe there should be a nick blacklist for the welcome redirect or something?

**Comment by flashcode** [comment]
There is the option `/set irc.look.notice_welcome_redirect`, maybe it could help?

**Comment by Mikaela** [comment]
I don't think so as it would make ChanServ and other bots go to server buffer instead of correct channels.

**Comment by wowaname** [comment]
The entire feature is a hack; making it more of a hack would be bad.

**Comment by Mikaela** [comment]
Making it not a hack on IRCv3-side https://github.com/ircv3/ircv3-specifications/issues/163.
